frontend/src/features/wizard/components/dashboard/CreateProfileSteps.scss

```diff
@@ -16,6 +16,47 @@
     width: 60%;
 
     .create-profile-section {
+      .Mui-completed path {
```

Comment:
Remove these additions (line 19-59) and replace the entire ".step" block (on line 62) with the following:

```
        .step {
          &__label {
            align-items: flex-start;

            &-container {
              text-align: start;
            }

            &--active {
              color: orbcStyles.$bc-background-blue;
              font-weight: bold;
            }

            &--completed {
              color: orbcStyles.$bc-green;
              font-weight: bold;
            }

            &--disabled {
              color: orbcStyles.$bc-text-box-border-grey;
              font-weight: bold;
            }
          }

          &__icon {
            overflow: visible;
            fill: orbcStyles.$white;
            stroke-width: 0.25rem;
            stroke: orbcStyles.$bc-text-box-border-grey;

            &--active {
              stroke: orbcStyles.$bc-background-blue;
            }

            &--completed {
              stroke: orbcStyles.$white;
              stroke-width: 0;
              fill: orbcStyles.$bc-green;
              transform: scale(1.2);
            }
          }

          &__step-number {
            display: none;
          }

          &__connector-line {
            border-width: 0.25rem;
          }

          &__connector {
            left: calc(-100% + 3em);
            right: calc(100% + 0.25em);
          }
        }
```

Reply:
Furthermore, in the files "CreateProfileSteps.tsx" and "ChallengeProfileSteps.tsx", replace the `<Stepper ... />` components with the following modifications:

```
          <Stepper
            className="stepper"
            activeStep={activeStep}
            alternativeLabel
            connector={
              <StepConnector
                className="step__connector"
                classes={{ line: "step__connector-line" }}
              />
            }
          >
            {steps.map((label) => (
              <Step className="step" key={label}>
                <StepLabel
                  className="step__label"
                  classes={{
                    labelContainer: "step__label-container",
                    active: "step__label--active",
                    disabled: "step__label--disabled",
                    completed: "step__label--completed",
                  }}
                  StepIconProps={{
                    className: "step__icon",
                    classes: {
                      text: "step__step-number",
                      active: "step__icon--active",
                      completed: "step__icon--completed",
                    }
                  }}
                >{label}</StepLabel>
              </Step>
            ))}
          </Stepper>
```

Reply:
Import any missing components (eg. StepConnector component) as needed in those files. The reasoning for these changes is the following:

1. It's generally not good practice to use default underlying .Mui-... classes for styling, since a) it's not clear as to which element you're referring to, and b) if for any reason the components need to be modified in the future, the order of those ".Mui-..." classes might change and complicated selectors (eg. "span.MuiStepLabel-label.Mui-disabled.MuiStepLabel-alternativeLabel") will most likely break. Hence it's better to override the nested Mui component classes by making use of the "classes" prop for certain components (like StepLabel, refer to MUI component API for more details).
2. Try to avoid directly referencing elements like "path" or "circle" or "text" when styling, whenever possible. It usually would suffice to apply css styling directly to the top-level svg element or its wrapper container.
3. A further improvement here would be to move the `<Stepper />` component into another file, and make use of that refactored component in both CreateProfileSteps.tsx and ChallengeProfileSteps.tsx. This is optional for the current PR, and can be done in the future.